test/utils/printer.js

```diff
@@ -70,18 +70,18 @@ const printEscrow = async ({ settings }, {testContext}, {tokenInfo, collateralTo
         return;
     }
 
-    const {
-        valueInToken,
-        valueInEth,
-    } = await escrow.calculateTotalValue();
-    const collateralBufferBytes32 = toBytes32(web3, platformSettingsNames.CollateralBuffer);
-    const collateralBufferSetting = await settings.getPlatformSetting(collateralBufferBytes32);
-    const collateralBuffer = parseInt(collateralBufferSetting.value);
-
-    console.log(`Collateral Buffer (%):             ${collateralBuffer} = ${collateralBuffer / 100} %`);
+    // const {
```

Comment:
Why is it commented? Please can we uncomment it?

Reply:
It throws an error when printing out the total escrow value when there are cTokens. This was fixed in my PR for updating the Aggregactors.